db/schema.rb

```diff
@@ -49,6 +49,15 @@
   add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true, using: :btree
   add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
 
+  create_table "donations", force: :cascade do |t|
+    t.string   "first_name"
+    t.string   "email"
+    t.integer  "amount"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string   "last_name"
+  end
```

Comment:
figaro to populate heroku keys?

On Thu, Jul 2, 2015 at 1:17 PM, David Molina <david.c.molina@gmail.com>
wrote:

> Trying it now.
>
> On Thu, Jul 2, 2015 at 1:09 PM, Eric Mckenna <notifications@github.com>
> wrote:
>
>> In db/schema.rb
>> <https://github.com/OperationCode/operationcode/pull/73#discussion_r33817388>
>> :
>>
>> > @@ -49,6 +49,15 @@
>> >    add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true, using: :btree
>> >    add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
>> >
>> > +  create_table "donations", force: :cascade do |t|
>> > +    t.string   "first_name"
>> > +    t.string   "email"
>> > +    t.integer  "amount"
>> > +    t.datetime "created_at", null: false
>> > +    t.datetime "updated_at", null: false
>> > +    t.string   "last_name"
>> > +  end
>>
>> probablay also need some sort of transaction id from the stripe response.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/OperationCode/operationcode/pull/73/files#r33817388>.
>>
>
>
>
> --
> David
> t. @davidcmolina <https://twitter.com/davidcmolina>
> http://davidmolina.github.io/
>



-- 
David
t. @davidcmolina <https://twitter.com/davidcmolina>
http://davidmolina.github.io/